ui/app/mirrors/[mirrorId]/page.tsx

```diff
@@ -115,77 +113,29 @@ export default async function ViewMirror({
         isNotPaused={isNotPaused}
       />
     );
-
-    return (
-      <LayoutMain alignSelf='flex-start' justifySelf='flex-start' width='full'>
-        <div
-          style={{
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-between',
-            paddingRight: '2rem',
-          }}
-        >
-          <Header variant='title2'>{mirrorId}</Header>
-          {actionsDropdown}
-        </div>
-        <CDCMirror
-          rows={rows}
-          createdAt={mirrorInfo?.created_at}
-          syncStatusChild={syncStatusChild}
-          status={mirrorStatus}
-        />
-      </LayoutMain>
-    );
   } else {
-    const runs = await prisma.qrep_partitions.findMany({
-      where: {
-        flow_name: mirrorId,
-        start_time: {
-          not: null,
-        },
-        rows_in_partition: {
-          not: 0,
-        },
-      },
-      orderBy: {
-        start_time: 'desc',
-      },
-    });
-
-    const partitions: QRepPartitionStatus[] = runs.map((run) => ({
-      partitionId: run.partition_uuid,
-      startTime: run.start_time,
-      endTime: run.end_time,
-      pulledRows: run.rows_in_partition,
-      syncedRows: Number(run.rows_synced),
-    }));
-
-    return (
-      <LayoutMain alignSelf='flex-start' justifySelf='flex-start' width='full'>
-        <div
-          style={{
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-between',
-            paddingRight: '2rem',
-          }}
-        >
-          <Header variant='title2'>{mirrorId}</Header>
-          <QRepStatusButtons mirrorId={mirrorId} />
-        </div>
-        <div>Status: {mirrorStatus.currentFlowState}</div>
-        <QrepGraph
-          syncs={partitions.map((partition) => ({
-            partitionID: partition.partitionId,
-            startTime: partition.startTime,
-            endTime: partition.endTime,
-            numRows: partition.pulledRows,
-          }))}
-        />
-        <br></br>
-        <QRepStatusTable flowJobName={mirrorId} partitions={partitions} />
-      </LayoutMain>
-    );
+    redirect(`/mirrors/status/qrep/${mirrorId}`);
```

Comment:
this redirect shouldn't happen, should be updating link instead of reverting change